src/conversion/definition-based-converters.js

```diff
@@ -0,0 +1,288 @@
+		.for( ...dispatchers )
+		.fromAttribute( attributeName )
+		.toElement( value => {
+			if ( value != attributeValue ) {
+				return;
+			}
+
+			return AttributeElement.createFromDefinition( viewDefinition );
+		} );
+}
+
+/**
+ * Helper for creating view to model converter from view to model attribute. It will convert also all matched view elements defined in
+ * `acceptAlso` property. The `model` property is used as model's attribute value to match.
+ *
+ * Conversion from model to view might be defined as simple one to one conversion:
+ *
+ *		viewToModelAttribute( 'bold', { model: true, view: 'strong' }, [ dispatcher ] );
+ *
+ * As a full-flavored definition:
+ *
+ *		viewToModelAttribute( 'fontSize', {
+ *			model: 'big',
+ *			view: {
+ *				name: 'span',
+ *				style: {
+ *					'font-size': '1.2em'
+ *				}
+ *			}
+ *		}, [ editor.data.viewToModel ] );
+ *
+ * or with `acceptAlso` property to match many elements:
+ *
+ *		viewToModelAttribute( 'fontSize', {
+ *			model: 'big',
+ *			view: {
+ *				name: 'span',
+ *				class: 'text-big'
+ *			},
+ *			acceptAlso: [
+ *				{ name: 'span', attribute: { 'data-size': 'big' } },
+ *				{ name: 'span', class: [ 'font', 'font-huge' ] },
+ *				{ name: 'span', style: { font-size: '18px' } }
+ *			]
+ *		}, [ editor.data.viewToModel ] );
+ *
+ * The above example will convert an existing view elements:
+ *
+ *		<p>
+ *			An example text with some big elements:
+ *			<span class="text-big>one</span>,
+ *			<span data-size="big>two</span>,
+ *			<span class="font font-huge>three</span>,
+ *			<span style="font-size:18px>four</span>
+ *		</p>
+ *
+ * into `fontSize` model attribute with 'big' value set so it will be represented:
+ *
+ *		<paragraph>
+ *			An example text with some big elements:
+ *			<$text fontSize="big>one</$text>,
+ *			<$text fontSize="big>two</$text>,
+ *			<$text fontSize="big>three</$text>,
+ *			<$text fontSize="big>four</$text>
+ *		</paragraph>
+ *
+ * @param {String} attributeName Attribute name to which convert.
+ * @param {module:engine/conversion/definition-based-converters~ConverterDefinition} definition A conversion configuration.
+ * @param {Array.<module:engine/conversion/viewconversiondispatcher~ViewConversionDispatcher>} dispatchers
+ */
+export function viewToModelAttribute( attributeName, definition, dispatchers ) {
+	const { model: attributeValue, viewDefinitions } = parseConverterDefinition( definition );
+
+	const converter = prepareViewConverter( dispatchers, viewDefinitions );
+
+	converter.toAttribute( () => ( {
+		key: attributeName,
+		value: attributeValue
+	} ) );
+}
+
+// Prepares a {@link module:engine/conversion/definition-based-converters~ConverterDefinition definition object} for building converters.
+//
+// @param {module:engine/conversion/definition-based-converters~ConverterDefinition} definition An object that defines view to model
+// and model to view conversion.
+// @returns {Object}
+function parseConverterDefinition( definition ) {
```

Comment:
This function gets JSON and return JSON, it parse nothing. It should be called "normaliseConverterDefinition".

Reply:
That's the word I was seeking then :)

Reply:
`normalizeConverterDefinition()` in AmE.